Settings/sparkle_settings.ini

```diff
@@ -1,6 +1,6 @@
 [general]
 objective = RUNTIME:PAR10
-configurator = smac_v2
+configurator = SMACv2
```

Comment:
Can we make this SMAC2? This is how we pronounce it when talking with other people about it

Reply:
Done